sdk/communication/communication-common/tests.yml

```diff
@@ -0,0 +1,29 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/communication-common"
+      ServiceDirectory: communication
+      CloudConfig:
+        Public:
+          SubscriptionConfigurations:
+            - $(sub-config-azure-cloud-test-resources)
+            - $(sub-config-communication-services-cloud-test-resources-common)
+            - $(sub-config-communication-services-cloud-test-resources-js)
+          MatrixFilters:
+            - DependencyVersion=^.+$  
+        PPE:
+          SubscriptionConfigurations:
+            - $(sub-config-communication-ppe-test-resources-common)
+            - $(sub-config-communication-ppe-test-resources-js)
+          MatrixFilters:
+            - DependencyVersion=^.+$  
```

Comment:
what's the `- DependencyVersion=^.+$  ` about?

Reply:
Matrix Filters is used to basically to filter out jobs.
- DependencyVersion=^.+$ --> means consider jobs for which dependency version is non-empty.

I will explain this in detail during our demo session.